crates/re_types_blueprint/.gitattributes

```diff
@@ -0,0 +1,129 @@
+# DO NOT EDIT! This file is generated by crates/re_types_builder/src/lib.rs
+
+.gitattributes linguist-generated=true
+src/archetypes/annotation_context.rs linguist-generated=true
+src/archetypes/arrows3d.rs linguist-generated=true
```

Comment:
interesting bug in codegen to put all types here!
doesn't need to be fixed in this pr since it doesn't break anything and probably always been like that